addons/epic-anchors/AnchorsPanel/TextureModulateButton.gd

```diff
@@ -2,8 +2,8 @@ tool
 extends TextureButton
 
 export(Color) var normal_col
-export(Color) var hover_col
 
+var hover_col := Color("f63c3c")
```

Comment:
Again, use a project setting here.

Reply:
These 2 are just default values in case something went wrong i guess, no need to worry about those

Reply:
Okay